### PR TITLE
8 header and footer

### LINKED ISSUE #8: Header and Footer

### PR DESCRIPTION
**Pull request by Mostafaelkhmisi**
Closed #8

### CODE REVIEW

File: src/assets/sass/website/app.scss [hunk 1]
```diff
@@ -11,7 +11,7 @@ body {
 
 .app-wrapper {
   height: 100%;
-  font-family: "Roboto";
+  font-family: Segoe UI;
```
Comment (engmagdy87) [inline src/assets/sass/website/app.scss:14]: enclose it by quotes

File: src/assets/sass/website/shared/footer.scss [hunk 2]
```diff
@@ -2,15 +2,206 @@
...
+$fontsize-paragraph:1.2rem;
+$clippath-botRight-leftTop:polygon(14% 0, 100% 0, 100% 85%, 86% 100%, 0 100%, 0 15%);
+$transition-5:all .5s ease-in-out;
+$transition-3:all .3s ease-in-out;
+
+
+.footer-section{
+    background-color: $dark-background;
+    padding-top: 175px!important;
+    .footer-container{
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: why u didn't apply BEM?

File: src/assets/sass/website/shared/footer.scss [hunk 3]
```diff
@@ -2,15 +2,206 @@
...
+            @include is-mobile{
+                font-size: .8rem;
+            }
+        }
+        i{
+            color: $header-item;
+            padding-right: 5px;
+        }
+        @include is-mobile {
+            max-width: 100%!important;
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: apply `max-width` without `important`

File: src/assets/sass/website/shared/footer.scss [hunk 4]
```diff
@@ -2,15 +2,206 @@
...
+            flex: 100%;
+            text-align: center;
+        }
+        @include is-tablet {
+            max-width: 100%!important;
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: apply `max-width` without `important`

File: src/assets/sass/website/shared/footer.scss [hunk 5]
```diff
@@ -2,15 +2,206 @@
...
+        }
+}
+
+
+.footer-socialmedia{
+    padding-bottom: 100px;
+
+
+    .footer-followus{
+        width: 108px!important;
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: apply `width` without `important`

File: src/assets/sass/website/shared/footer.scss [hunk 6]
```diff
@@ -2,15 +2,206 @@
...
+        padding-bottom: 25px;
+        font-family: Segoe UI !important;
+        font-weight: bolder;
+        font-size: 20px;
+            &::after{
+              content: "";
+              position: absolute;
+              bottom: 0;
+              top: 28px;
+              left: 0px!important;
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: apply `left` without `important`

File: src/assets/sass/website/shared/footer.scss [hunk 7]
```diff
@@ -2,15 +2,206 @@
...
+.footer-socialmedia{
+    padding-bottom: 100px;
+
+
+    .footer-followus{
+        width: 108px!important;
+        position: relative;
+        color: $header-item;
+        padding-bottom: 25px;
+        font-family: Segoe UI !important;
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: remove `font-family`

File: src/assets/sass/website/shared/footer.scss [hunk 8]
```diff
@@ -2,15 +2,206 @@
...
+        font-weight: bolder;
+        font-size: 20px;
+            &::after{
+              content: "";
+              position: absolute;
+              bottom: 0;
+              top: 28px;
+              left: 0px!important;
+              height: 4px;
+              width: 56%!important;
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: remove all upcoming `important`

File: src/assets/sass/website/shared/footer.scss [hunk 9]
```diff
@@ -2,15 +2,206 @@
...
+        }
+        @include is-tablet {
+            margin: auto!important;
+        }
+        @include is-desktop {
+            margin: auto!important;
+        }
+    }
+    
+    .socialMedia{
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer.scss]: just use `justify-content: center;` if it used in all mixins with the same value and remove all mixins

File: src/assets/sass/website/shared/footer1.scss [hunk 10]
```diff
@@ -0,0 +1,16 @@
+@import "../mixins.scss";
```
Comment (engmagdy87) [inline src/assets/sass/website/shared/footer1.scss]: bad naming for the file
change it to a meaningful name

File: src/website/shared/Footer.vue [hunk 11]
```diff
@@ -1,157 +1,75 @@
...
-        <img
-          class="footer-wrapper__logo"
-          src="/website/img/footer/worx.png"
-          alt="worx"
-          @click="redirectTo('https://theworx-emc.com/')"
-        />
-      </div>
-    </div>
+      <!-------------  Home Footer ------------------->
+<div class="footer-section">
```
Comment (engmagdy87) [inline src/website/shared/Footer.vue:3]: remove unnecessary comments

File: src/website/shared/Footer1.vue [hunk 12]
```diff
@@ -0,0 +1,157 @@
+<template>
```
Comment (engmagdy87) [inline src/website/shared/Footer1.vue]: bad file name

File: src/website/shared/Footer1.vue [hunk 13]
```diff
@@ -0,0 +1,157 @@
...
+        <img
+          class="footer-wrapper__logo"
+          src="/website/img/footer/worx.png"
+          alt="worx"
+          @click="redirectTo('https://theworx-emc.com/')"
+        />
+      </div>
+    </div>
+
+    <!-- <div class="footer-wrapper__large-footer">
```
Comment (engmagdy87) [inline src/website/shared/Footer1.vue]: remove comments

File: src/website/shared/Footer1.vue [hunk 14]
```diff
@@ -0,0 +1,157 @@
...
+
+<script>
+export default {
+  // props: ["sponsors", "events", "showFooter"],
+  methods: {
+    redirectTo(url) {
+      if (url.includes("http")) window.open(url, "_blank");
+      else window.open(`http://${url}`, "_blank");
+    }
+    // redirectEventTo(event) {
```
Comment (engmagdy87) [inline src/website/shared/Footer1.vue]: remove comments

**Review by engmagdy87 — CHANGES_REQUESTED** [review]
why did u add all files of `About` in this branch?
the branch should have its related work only

File: src/assets/sass/website/shared/footer.scss (hunk 9 above)
Comment (Mostafaelkhmisi) [reply src/assets/sass/website/shared/footer.scss]: its not in the center in the big screens only in the desktop and below

File: src/assets/sass/website/shared/footer.scss (hunk 7 above)
Comment (Mostafaelkhmisi) [reply src/assets/sass/website/shared/footer.scss]: u mean remove !important ?

File: src/assets/sass/website/shared/footer.scss (hunk 9 above)
Comment (engmagdy87) [reply src/assets/sass/website/shared/footer.scss]: put it outside and change `justify-content` NOT to be `center` in the big screens

File: src/assets/sass/website/shared/footer.scss (hunk 7 above)
Comment (engmagdy87) [reply src/assets/sass/website/shared/footer.scss]: yes

**Review by engmagdy87 — APPROVED** [review]

**Pull request merged**